unesp-dracena.csl

```diff
@@ -0,0 +1,589 @@
+            <text macro="translator" suffix=". "/>
+            <text macro="edition" suffix=". "/>
+            <text variable="version" suffix=". "/>
+            <text macro="publisher" suffix=". "/>
+            <text macro="locators"/>
+            <text variable="note" suffix=". "/>
+            <text macro="access"/>
+          </group>
+        </else-if>
+        <else-if type="chapter">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text term="in" text-case="capitalize-first" suffix=": "/>
+            <text macro="container-contributors" suffix=". "/>
+            <text macro="container-title" suffix=". "/>
+            <text variable="collection-title"/>
+            <text macro="translator" suffix=". "/>
+            <text macro="edition" suffix=". "/>
+            <text macro="publisher" suffix=". "/>
+            <text macro="locators" suffix=". "/>
+          </group>
+        </else-if>
+        <else-if type="article-magazine article-journal" match="any">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="container-title" suffix=", "/>
+            <text macro="collection-title" suffix=". "/>
+            <text macro="place"/>
+            <text macro="edition" suffix=", "/>
+            <text macro="locators" suffix=", "/>
+            <text macro="issued" suffix=". "/>
+            <text macro="access"/>
+          </group>
+        </else-if>
+        <else-if type="article-newspaper" match="any">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="container-title" suffix=", "/>
+            <text macro="place"/>
+            <text macro="locators" suffix=", "/>
+            <text macro="issued-legislation" suffix=". "/>
+            <text macro="section" suffix=". "/>
+            <text macro="access"/>
+          </group>
+        </else-if>
+        <else-if type="thesis">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="issued-year" suffix=". "/>
+            <text macro="genre" suffix=" - "/>
+            <text variable="publisher" suffix=", "/>
+            <text macro="place"/>
+            <text macro="issued" suffix="."/>
+            <text macro="access"/>
+          </group>
+        </else-if>
+        <else-if type="manuscript">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="edition" suffix=". "/>
+            <text macro="place" suffix=", "/>
+            <text macro="issued" suffix=". "/>
+            <text macro="access" suffix=". "/>
+            <text macro="archive" suffix=". "/>
+          </group>
+        </else-if>
+        <else-if type="webpage">
```

Comment:
we usually group webpages and post-weblogs together as they're both web sources.